paws.common/R/config.R

```diff
@@ -150,29 +150,16 @@ get_env <- function(variable) {
   return("")
 }
 
-# Get the value of an OS environment variable.
-# NOTE: Does not work on Windows.
-get_os_env <- function(var) {
-  if (.Platform$OS.type == "unix") {
-    value <- system(sprintf("echo $%s", var), intern = T)
-  } else {
-    value <- "" # Not implemented on Windows.
-  }
+# Get the name of the IAM role from the instance metadata.
+get_iam_role <- function() {
```

Comment:
Moved `get_iam_role()` to `config.R` from `credential_providers.R`, given parallel to `get_instance_metadata()` and similar functions in the former (e.g., pulling from env variables)